cpp/src/io/csv/csv_reader.cu

```diff
@@ -1204,7 +1209,7 @@ gdf_error launch_dataTypeDetection(
 		d_columnData
 	);
 
-	CUDA_TRY( cudaGetLastError() );
+	CUDA_TRY( cudaGetLastError());
```

Comment:
Since the previous PR was accepted, merge conflicts resulted in having to manually copy the previous changes back, hence the unintentional extra spaces. Will proceed to fix these trivial things.